docus/content/rules/process.md

```diff
@@ -23,25 +22,19 @@ All staff should be intimately familiar with the [Rules](/rules)
 
 Moderators are able to take action on any reported or observed user account in violation of our rules.
 
-### Directors
+### Administrators
 
-Directors have the ability to de-federate entire instances for continued violation or bad actions against our members, or our infrastructure. They also act as senior moderators and community leaders.
+Administrators have the ability to de-federate entire instances for continued violation or bad actions against our members, or our infrastructure. They also act as senior moderators and community leaders.
 
-They can also perform additional actions on user accounts beyond those of a normal moderator.
-
-### Administrator
-
-The administrator would really like to focus on things other than daily moderation of user activities. With the exception of appeals, most moderation activities are undertaken by Moderators or Directors.
-
-The Administrator is **the final authority** on any appeals, moderation issues or rule interpretations.
+The Administrators are **the final authority** on any appeals, moderation issues or rule interpretations.
 
 ## Reports
 
-When a post is reported, all members of the staff will receive an email alert. There is also an alert message posted to the #moderation channel of the backend Slack. Both alerts include links to the report in the Mastodon UI.
+When a post is reported, all members of the staff will receive an email alert. There is also an alert message posted to the "vmst.io Moderation" room in Matrix. Both alerts include links to the report in the Mastodon UI.
 
 Reports about members of the staff cannot be seen by that staff member in the Mastodon UI.
 
-For whatever action you take on a report, please post to the thread in Slack as an indicator that you’ve followed up. If you need to loop in another staffer for an assist do so via @mention within Slack.
+For whatever action you take on a report, please post to the thread in Matrix as an indicator that you’ve followed up. If you need to loop in another staffer for an assist do so via @mention within Matrix.
```

Comment:
This seems oddly placed here. We've gone from talking to general vmst.io users to Mods specifically.
Perhaps a separate page or location for "how to be a vmst.io mod"?

Reply:
The page is meant to be our guidance, made public.